pipe_tools/timestamp.py

```diff
@@ -139,24 +137,6 @@ def rfc3339strFromTimestamp(ts):
 def timestampFromRfc3339str(s):
     return timestampFromUdatetime(udatetime.from_string(s))
 
-def str2date(s):
-    """
-    :param s date in string expresed with the format AIRFLOW_DATE
-    :type s string
-    :return datetime representing the string s.
-    """
-    return datetime.strptime(s, AIRFLOW_DATE)
-
-def daterange(start_date, end_date):
```

Comment:
Why did you remove these method? How are you sure it isn't being used in any dataflow process?

Reply:
I created only for BigqueryOperator purposes.